lr-cli/cortx_setup/commands/storage/config.py

```diff
@@ -81,12 +83,14 @@ class StorageEnclosureConfig(Command):
         'controller_type': {
             'type': str,
             'optional': True,
+            'choices': ['gallium', 'indium', 'virtual'],
```

Comment:
Avoid using 'gallium', 'indium'.
Instead we could use 'choices': ['<one of Seagate controller names>', 'virtual']

Reply:
These choices are mentioned in the factory confluence page for the controller_type argument.

Reply:
Ok, then we have no choice :(